R/dimensions.R

```diff
@@ -101,3 +102,19 @@ pitch_international <- list(
   origin_x = 0,
   origin_y = 0
 )
+
+#' @rdname pitch_tracab
+#' @export
+# ChyronHego Tracab: only length & width provided in metadata.
+# Source: official documentation.
+# This solution takes the remaining dimensions from the international pitch.
+make_pitch_tracab <- function(length=105, width=68){
+  pitch_tracab <- lapply(pitch_international, function(x){100.0*x}) # cm
```

Comment:
It's purely down to personal preference, but you can omit the braces for an inline function definition ([e.g.](https://style.tidyverse.org/syntax.html#inline-statements)). In the absence of a nice lambda syntax in base R, I think it makes them stand out a little more. Equally, if you prefer this style, then it's fine with me. Both are valid and readable.